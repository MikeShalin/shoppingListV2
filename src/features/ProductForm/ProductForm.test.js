import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import ProductFormComponent from './index'
import ProductFormModel from 'Models/ProductForm'

const productForm = ProductFormModel.create({
  productName: '',
})

describe('ProductForm component', () => {
  it('matches early snapshots', () => {
    const props = {
      handleChange: () => false,
      productForm: {
        onSubmit: () => true,
        productName: '',
      },
    }
    const tree = shallow(
      <ProductFormComponent.wrappedComponent productForm={productForm} {...props} />,
    )
    expect(shallowToJson(tree)).toMatchSnapshot()
  })
})