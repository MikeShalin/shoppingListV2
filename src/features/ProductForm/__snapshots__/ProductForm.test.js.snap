// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductForm component matches early snapshots 1`] = `
<FormExampleForm
  handleChange={[Function]}
  productForm={
    Object {
      "onSubmit": [Function],
      "productName": "",
    }
  }
/>
`;
